docs/other/configuration-with-osg-configure.md

```diff
@@ -0,0 +1,550 @@
+Configuration with OSG-Configure
+================================
+
+-   [About this document](#about-this-document)
+-   [Invocation and script usage](#invocation-and-script-usage)
+-   [Syntax and layout](#syntax-and-layout)
+-   [Configuration sections](#configuration-sections)
+    -   Job managers (batch systems):
+        -   [Bosco](#bosco)
+        -   [Condor](#condor)
+        -   [LSF](#lsf)
+        -   [PBS](#pbs)
+        -   [SGE](#sge)
+        -   [Slurm](#slurm)
+    -   Monitoring/reporting:
+        -   [Gratia](#gratia)
+        -   [Info Services](#info-services)
+        -   [RSV](#rsv)
+        -   [Subcluster / Resource Entry](#subcluster-resource-entry)
+    -   [Gateway](#gateway)
+    -   [Local Settings](#local-settings)
+    -   [Misc Services](#misc-services)
+    -   [Site Information](#site-information)
+    -   [Squid](#squid)
+    -   [Storage](#storage)
```

Comment:
This seems superfluous with the ToC on the left, no?

Reply:
I think the categories help